pom.xml

```diff
@@ -16,13 +16,17 @@
 	<description>Demo project for Spring Boot</description>
```

Comment:
please look at how the upgrade was done in EBL and BKG (beta2 branch). If you set dcsa-bom as parent you can remove a lot of the config from here as it then gets it from dcsa-bom

Reply:
Done